pxr/usdImaging/usdviewq/appController.py

```diff
@@ -982,6 +982,13 @@ def __init__(self, parserData, resolverContextFn):
             self._ui.actionShow_Abstract_Prims.triggered.connect(
                 self._toggleShowAbstractPrims)
 
+            # nv begin #prim-display-name
```

Comment:
These comment blocks will get removed on the next push. Thanks for the quick turnaround, Ed!

Reply:
Removed on latest commit